src/components/Button.astro

```diff
@@ -6,7 +6,7 @@ const { variant = 'primary' } = Astro.props;
 ---
 
 <span class:list={[`link pixel variant-${variant}`, className]} {style}>
-	<a {href}>
+	<a {href} target="_blank">
```

Comment:
In general I’d advise against opening in new windows. It enforces behaviour on users which they can’t easily opt out of and may not expect, whereas it is relatively simple to opt _in_ to opening in a new tab/window if you want it.

This article is helpful in listing some more reasons to avoid this 😅 
https://css-tricks.com/use-target_blank/

Reply:
Right, I think the overall advice in that article is don't change default behaviour. A user can use cmd/ctrl + click to open in a new tab (or right click etc) if that's what they want but they probably won't (and can't) preemptively make sure to open in the current tab if _that's_ what they want.

Reply:
Gotcha, this makes sense.